src/aws.ts

```diff
@@ -94,14 +95,18 @@ export class AWS {
         )
     }
 
-    public async uploadFile(bucket: string, key: string, body: ReadStream, objectMimeType: string) {
+    public async uploadFile(bucket: string, key: string, body: ReadStream, objectMimeType: string, objectACL?: ObjectCannedACL) {
         const uploadParams: PutObjectRequest = {
             Body: body,
             Bucket: bucket,
             ContentType: objectMimeType,
             Key: key,
         }
 
+        if (objectACL) {
+            uploadParams.ACL = objectACL
```

Comment:
Why do you need this check, what would happen if `objectACL` is undefined and `uploadParams` object has `ACL` value of undefined?

I think that would be fine